product_attribute_code/product_view.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  Module for OpenERP
+  Copyright (C) 2015 Akretion (http://www.akretion.com). All Rights Reserved
+  @author Benoît GUILLOT <benoit.guillot@akretion.com>
+  The licence is in the file __openerp__.py
+-->
+
+<openerp>
+    <data>
+        <!-- INHERITED VIEW FOR THE OBJECT : product_attribute -->
+
+        <record id="attr_tree_view" model="ir.ui.view">
+            <field name="name">product_attribute_code.attribute.tree</field>
+            <field name="model">product.attribute</field>
+            <field name="inherit_id" ref="product.attribute_tree_view"/>
+            <field name="arch" type="xml">
+                <field name="name" position="before">
+                    <field name="sequence"/>
```

Comment:
@ahilali Hi,

I've taken another look but I still can't test it against a real dataset because of the circular issue.  I'm not sure how to solve it.  Maybe it needs a configuration wizard or something to set things up if there are existing attribute lines.  

Have added a couple of new comments of things I saw when reviewing the code more closely.